.github/workflows/ci.yml

```diff
@@ -92,3 +92,62 @@ jobs:
 
       - name: Show Cache
         run: du -h ${{ github.workspace }}/.cache/* || true
+
+  release-pr:
+    runs-on: ubuntu-latest
+    needs: [test]
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          release-type: node
+          package-name: ipfs-companion
+          changelog-notes-type: github
+          command: release-pr
+
+  release-assets:
+    runs-on: ubuntu-latest
+    needs: [test]
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v1
+
+      - name: Install Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 18
+
+      - name: Restore .cache
+        uses: actions/cache@v2
+        id: cache
+        with:
+          path: ${{ github.workspace }}/.cache
+          key: ${{ runner.os }}-${{ hashFiles('package*json', 'package-lock.json', '*config.js') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ hashFiles('package*json', 'package-lock.json', '*config.js') }}
+            ${{ runner.os }}-
+
+      - name: Restore node_modules
+        id: yarn-cache
+        uses: actions/cache@v2
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
+
+      - name: Install dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: npm run ci:install
+
+      - name: Build Stable Release
+        run: npm run ci:build:stable
+
+      - name: Release
+        uses: softprops/action-gh-release@v1
```

Comment:
:warning:  @whizzzkid  any reason why we are not reusing `google-github-actions/release-please-action` here with `command` set to `github-release`? 

Not a blocker, but `softprops/action-gh-release`  being a personal account of an individual has a higher risk of being compromised, and that could inject code that would ship to our users.

If we have to use it, please pin it to a specific revision before merging this PR:

```suggestion
        uses: softprops/action-gh-release@de2c0eb89ae2a093876385947365aca7b0e5f844
```

For future reference, remember to add to the safelist at https://github.com/ipfs/ipfs-companion/settings/actions:
> ![2023-01-31-231838_819x604_scrot](https://user-images.githubusercontent.com/157609/215896607-0af35cd5-c6b6-4432-940b-902cd4550944.png)